tests/prague/eip7692_eof_v1/eip7620_eof_create/test_subcontainer_validation.py

```diff
@@ -572,14 +565,72 @@ def test_wide_container(eof_test: EOFTestFiller, width: int, exception: EOFExcep
                         + Op.POP
                         + Op.STOP
                     ),
-                    Section.Container(Container(sections=[Section.Code(Op.INVALID)])),
+                    Section.Container(Container.Code(Op.INVALID)),
                 ],
                 expected_bytecode="""
                 ef0001010004020001000b0300010014040000000080000436600060ff6000ec005000ef000101000402
                 000100010400000000800000fe""",
             ),
             id="EOF1_eofcreate_valid_0",
         ),
+        pytest.param(
+            Container(
+                sections=[
+                    Section.Code(Op.PUSH1[0] + Op.RJUMP[0] + Op.STOP),
+                    Section.Container(Container.Code(Op.INVALID)),
+                ],
+                expected_bytecode="""
+                ef00010100040200010006030001001404000000008000016000e0000000ef0001010004020001000104
+                00000000800000fe""",
+                # Originally this test was "valid" because it was created
+                # before "orphan subcontainer" rule was introduced.
+                validity_error=EOFException.ORPHAN_SUBCONTAINER,
+            ),
+            id="EOF1_embedded_container_0",
```

Comment:
I think these IDs should be a bit more explicative about what the test is trying to verify.

"EOF1_embedded_container_0" -> "orphan_subcontainer"
"EOF1_embedded_container_1" -> "orphan_subcontainer_with_incomplete_data_section"
"EOF1_embedded_container_2" -> "orphan_subcontainer_with_data_section"
"EOF1_embedded_container_3" -> "invalid_magic_subcontainer"

Just to make debugging easier for clients IMO.

Reply:
I used the ids of the original tests so you can compare if the bytecode is identical... but I can change them.

Reply:
Renamed.